app/core/src/main/java/com/teamg/tourdeshot/core/model/LocalWithDistance.java

```diff
@@ -0,0 +1,11 @@
+package com.teamg.tourdeshot.core.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class LocalWithDistance {
+
+    private Long id;
```

Comment:
Maybe LocalWithDistance should consist of Local instance and distance field? Then you could inject LocalMapper instance into LocallWithDistanceMapper and use it to map Local